service/src/setup.rs

```diff
@@ -0,0 +1,197 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+	Copyright (C) 2017-2019 Baidu, Inc. All Rights Reserved.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::error::{Error, ServiceResult};
+use codec::Encode;
+use itp_enclave_api::{enclave_base::EnclaveBase, Enclave};
+use itp_settings::files::{
+	LAST_SLOT_BIN, LIGHT_CLIENT_DB, SHARDS_PATH, SHIELDING_KEY_FILE, SIDECHAIN_STORAGE_PATH,
+	SIGNING_KEY_FILE,
+};
+use itp_types::ShardIdentifier;
+use log::*;
+use std::{fs, fs::File, path::Path};
+
+/// Purge all worker files from the current working directory (cwd).
+pub(crate) fn purge_files_from_cwd() -> ServiceResult<()> {
+	let current_directory = std::env::current_dir().map_err(|e| Error::Custom(e.into()))?;
+	println!("[+] Performing a clean reset of the worker");
+
+	println!("[+] Purge any previous files");
```

Comment:
```suggestion
	println!("[+] Purging all state files possibly left from a previous run");
```

Reply:
I'll update that message 👍 Is it okay for you if I leave out the `state` part? Since it's not only state files, but also keys

Reply:
Yes, I can life with that ;)